webview/src/stories/Table.stories.tsx

```diff
@@ -79,6 +80,9 @@ WithNoRunningExperiments.args = {
   }
 }
 
+export const WithAllDataTypes = Template.bind({})
+WithAllDataTypes.args = { tableData: dataTypesTableData }
```

Comment:
[Q] Can you add the boolean tooltip into this test with the `play` functionality?